versions.json

```diff
@@ -2,85 +2,37 @@
   "design-system": [
     "7.0.0-beta.2",
     "7.0.0-beta.1",
-    "6.1.2",
+    "6.1.3",
     "6.0.1",
-    "6.0.0-beta.3",
-    "6.0.0-beta.2",
     "5.0.2",
-    "5.0.1",
-    "5.0.0",
-    "5.0.0-beta.1",
     "4.0.4",
-    "4.0.2",
-    "4.0.1",
-    "4.0.0",
-    "4.0.0-beta.2",
     "3.8.0",
-    "3.8.0-beta.2",
     "3.7.0",
-    "3.7.0-beta.3",
-    "3.7.0-beta.2",
-    "3.7.0-beta.1",
     "3.6.0",
-    "3.6.0-beta.3",
-    "3.6.0-beta.2",
-    "3.6.0-beta.1",
     "3.4.0"
   ],
   "ds-medicare-gov": [
     "9.0.0-beta.2",
     "9.0.0-beta.1",
```

Comment:
Do we need both betas?

Reply:
I'll remove both since we're about to do a release